src/components/GameView/GameCard.vue

```diff
@@ -6,6 +6,8 @@
       glasses: isGlasses,
       jack: isJack,
       frozen: isFrozen,
+      threesPlayer: isThrees && playerChoosingFromScrap,
+      threesOpponent: isThrees && !playerChoosingFromScrap
```

Comment:
Let's make computed properties for the specific classes and just refer to them here like
```suggestion
      threesPlayer: isPlayerThreeTarget,
      threesOpponent: isOpponentThreeTarget
```

Then in our computed props we can have

```
computed: {
  isPlayerThreeTarget() {
    return this.isThreeTarget && this.playerChoosingFromScrap
  },
  isOpponentThreeTarget() {
    return this.isThreeTarget && !this.playerChoosingFromScrap;
  },
}
```